src/components/modals/PasswordReminderModal.tsx

```diff
@@ -80,11 +80,13 @@ export class PasswordReminderModalComponent extends React.PureComponent<Props, S
 
     return (
       <ModalUi4 bridge={bridge} title={lstrings.password_reminder_remember_your_password} onCancel={this.handleCancel}>
-        <ScrollView style={{ maxHeight: theme.rem(9) }}>
+        <ScrollView style={{ maxHeight: theme.rem(9), marginTop: theme.rem(1) }}>
```

Comment:
If you restore the margins on `ModalMessage`, you remove / reduce this `marginTop`.

Reply:
We also don't need this scroll anymore. It looks fine even on a small Android screen